code/clue.js

```diff
@@ -1,59 +1,237 @@
+  occupation: 'Monkey',
+  favouriteFood: 'Banana'
+};
 
-const mrGreen = {
-  firstName: 'Jacob',
+const Ross = {
+  firstName: 'Ross',
+  lastName: 'Geller',
+  description: 'A palaeontologist working at the Museum of Natural History and in love with Rachel.',
+  age: 27,
+  image: 'assets/ross.png',
+  occupation: 'Palaeontologist',
+  favouriteFood: 'Muffin'
+};
+
+const Chandler = {
+  firstName: 'Chandler',
+  lastName: 'Muriel Bing',
+  description: 'Making jokes whe he feels himself uncomfortable and loves Monica.',
+  age: 27,
+  image: 'assets/chandler.png',
+  occupation: 'Transponster',
+  favouriteFood: 'Pizza'
+};
+
+const Monica = {
+  firstName: 'Monica',
+  lastName: 'Geller',
+  description: 'Monica was overweight as a child.',
+  age: 26,
+  image: 'assets/monica.png',
+  occupation: 'Chef',
+  favouriteFood: 'Kiwi-lime pie'
+};
+
+const Rachel = {
+  firstName: 'Rachel',
   lastName: 'Green',
-  color: 'green',
-  description: 'He has a lot of connections',
-  age: 45,
-  image: 'assets/green.png',
-  occupation: 'Entrepreneur'
+  description: 'A fashion enthusiast and Monica Gellers best friend from childhood.',
+  age: 25,
+  image: 'assets/rachel.png',
+  occupation: 'Waitress',
+  favouriteFood: 'Trifle'
+};
+
+const Phoebe = {
+  firstName: 'Phoebe',
+  lastName: 'Buffay',
+  description: 'She writes and sings her own strange songs, accompanying herself on the guitar.',
+  age: 27,
+  image: 'assets/phoebe.png',
+  occupation: 'A self-taught musician.',
+  favouriteFood: 'Nothing which had a name'
+};
+
+const Hamburger = {
+  name: 'Hamburger',
+  taste: 'Taste: 9',
+  image: 'assets/hamburger.jpg'
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const RachelTrifle = {
+  name: "Rachel's Trifle",
+  taste: 'Taste: 2',
+  image: 'assets/trifle.jpg'
+}
 
-const rope = {
-  name: 'rope',
-  weight: 10
+const Turkey = {
+  name: 'Turkey',
+  taste: 'Taste: 10',
+  image: 'assets/turkey.jpeg'
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const RedBagel = {
+  name: 'Red Bagel',
+  taste: 'Taste: 6',
+  image: 'assets/bagel.jpg'
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const Pizza = {
+  name: 'Pizza',
+  taste: 'Taste: 7',
+  image: 'assets/pizza.png'
+}
 
-const suspects = [
-  mrGreen,
-  mrsWhite
-  // ...  and the rest
-]
+const Muffin = {
+  name: 'Muffin',
+  taste: 'Taste: 4',
+  image: 'assets/muffin.jpg'
+}
+
+const KiwiPie = {
+  name: 'Kiwi Pie',
+  taste: 'Taste: 3',
+  image: 'assets/pie.jpg'
+}
+
+const Sandwich = {
+  name: 'Sandwich',
+  taste:'Taste: 5',
+  image: 'assets/sandwich.jpg'
+}
 
-const weapons = []
+const MonicaDiningRoom = {
+  name: "Monica's Dining Room",
+  image: 'assets/mdr.jpg'
+}
+
+const RossApartment = {
+  name: "Ross' Apartment",
+  image: 'assets/ra.jpg'
+}
 
-const rooms = []
+const ChandlerDiningRoom = {
+  name: "Chandler's Dining Room",
+  image: 'assets/cdr.jpg'
```

Comment:
Not a huge problem but remember that Maks told us to pick either " or ' to write in our code to make it more clean using one of them. 😁

Reply:
Yes, you're totally right, I am going to fix it! Thanks a lot!!!

Reply:
Exactly :)